Dockerfile.app

```diff
@@ -7,16 +7,16 @@ RUN curl -f https://get.pnpm.io/v6.16.js | node - add --global pnpm
 
 WORKDIR /web
 
-# copy all dependency declerations inside the container
+# copy all dependency declarations inside the container
 COPY package*.json .
 COPY pnpm*.yaml .
-COPY app/package*.json ./app/
+COPY tasks/package*.json ./tasks/
 COPY lib/package*.json ./lib/
 
-# install depedencies
+# install dependencies
 RUN pnpm install --frozen-lockfile
```

Comment:
you need to copy the whole `lib/` folder before doing `pnpm install` as `lib/` is a dependency of `tasks/`